packages/edit-site/src/components/start-page-options/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	store as blockEditorStore,
+	StarterPatternsModal,
+} from '@wordpress/block-editor';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import { store as preferencesStore } from '@wordpress/preferences';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../store';
+
+export default function StartPageOptions() {
```

Comment:
Why duplicate this here. I thought that personally we should just move the whole component to the "editor" package (without extracting things to block-editor) and render the component directly within `EditorProvider`. (And yes, we need to relax the change to only check for "content" and not "content+title")

Reply:
@youknowriad Back when I worked on this there were a bunch of inconsistencies between the editors (I don't think there was even a consistent to get the current page id), so I extracted all the common stuff to a new `StarterPatternsModal` component. Those inconsistencies might not exist any more today with all the unification efforts, so using one component could be a possibility.

I won't have any bandwidth to work on this till post WordPress 6.5, so feel free to take over.